content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -80,6 +80,14 @@ Used on: All Objects
 
 This annotation is used for describing specific behaviour of given object.
 
+## kubernetes.io/enforce-mountable-secrets {#enforce-mountable-secrets}
+
+Example: `kubernetes.io/enforce-mountable-secrets: "true"`
```

Comment:
nit: Should the value use double quotes? Instead:
Example: `kubernetes.io/enforce-mountable-secrets: true`

Reply:
Maybe, maybe not. The keys and values of annotations must be strings. If the double quotes are omitted, k8s may treat this as a boolean? Not sure.

Reply:
value should be double quoted as a string

Reply:
👍